app/api/helpers/permissions.py

```diff
@@ -87,5 +87,113 @@ def decorated_function(*args, **kwargs):
 
     return decorated_function
 
+@second_order_decorator(jwt_required)
+def is_organizer(f):
+    """
+    Allows only Organizer to access the event resources.
+    :param f:
+    :return:
+    """
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        user = current_identity
+        if kwargs['event_id'] and user.is_organizer(kwargs['event_id']):
+            return f(*args, **kwargs)
+        return ForbiddenError({'source': ''}, 'Access Forbidden').respond()
+
+    return decorated_function
+
+@second_order_decorator(jwt_required)
+def is_coorganizer(f):
+    """
+    Allows Organizer and Co-organizer to access the event resources.
+    :param f:
+    :return:
+    """
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        user = current_identity
+        if kwargs['event_id']:
+            event_id = kwargs['event_id']
```

Comment:
In the above function you didn't assign it to a separate variable. Here you did. Is there any specific reason for doing this?